Gemfile.lock

```diff
@@ -105,19 +105,19 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (0.9.2)
-    mimemagic (0.3.3)
+    mimemagic (0.3.4)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.13.0)
-    msgpack (1.3.1)
+    minitest (5.14.0)
+    msgpack (1.3.3)
     nio4r (2.5.2)
-    nokogiri (1.10.7)
+    nokogiri (1.10.8)
       mini_portile2 (~> 2.4.0)
-    popper_js (1.14.5)
-    public_suffix (4.0.2)
+    popper_js (1.16.0)
```

Comment:
### [popper_js](https://github.com/glebm/popper_js-rubygem)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.14.5 → 1.16.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **3** commits. See the full changes on [the compare page](https://github.com/glebm/popper_js-rubygem/compare/v1.14.5...v1.16.0).

These are the individual commits:
- (b2cdf94) [Update to 1.14.7](https://github.com/glebm/popper_js-rubygem/commit/b2cdf94584a034360ea9221e1f399cb0fccb639c)
- (acddfd9) [Update to 1.15.0](https://github.com/glebm/popper_js-rubygem/commit/acddfd9695a508b143c03bbe4ad9b61a24240fd7)
- (e82ae84) [Update to 1.16.0](https://github.com/glebm/popper_js-rubygem/commit/e82ae8406f4286139d6ac89e8fd58cca548a1026)


</details>